### PR TITLE
make cell boundary a keyword

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

**Comment by fredrikekre** [comment]
Some missing definitions in generate_grid

**Comment by KristofferC** [comment]
Yep,  I changed some more stuff.

I made the element sets strings because that's sort of what they are.  Just arbitrary names, symbols are usually when you have one from a finite set. I also tweaked the accessor functions a bit. Opinions?

File: src/grid.jl [hunk 1]
```diff
@@ -8,7 +8,8 @@ export Line, QuadraticLine,
 
 # Grid utilities
 export getcells, getncells, getnodes, getnnodes, getcelltype,
-       getcellset, getnodeset, getcellboundaryset, getcoordinates
+       getcellset,  getnodeset, getcellboundaryset, getcoordinates,
```
Comment (fredrikekre) [inline src/grid.jl]: white space?!

File: src/grid.jl [hunk 2]
```diff
@@ -69,36 +70,54 @@ immutable Grid{dim, N, T <: Real}
...
+                         nodesets::Dict{String, Vector{Int}}=Dict{String, Vector{Int}}(),
+                         cellboundarysets::Dict{String, Vector{Int}}=Dict{String, Vector{Int}}())
     return Grid(cells, nodes, cellboundaries, cellsets, nodesets, cellboundarysets)
 end
 
 ##########################
 # Grid utility functions #
 ##########################
 @inline getcells(grid::Grid) = grid.cells
+@inline getcells(grid::Grid, v::Vector{Int}) = grid.cells[v]
```
Comment (fredrikekre) [inline src/grid.jl]: This was nicer like this.

File: test/test_VTK.jl [hunk 3]
```diff
@@ -1,7 +1,7 @@
 using SHA
 @testset "VTK" begin
     OVERWRITE_CHECKSUMS = false
-    checksums_file = "checksums.sha1"
+    checksums_file = joinpath(dirname(@__FILE__), "checksums.sha1")
```
Comment (fredrikekre) [inline test/test_VTK.jl]: Nice, I always `cd`'d myself into the test directory :p

**Comment by KristofferC** [comment]
Thx

**Pull request merged**